assembly-multiuser/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -120,3 +124,18 @@ che.keycloak.github.endpoint=NULL
 
 #     The number of seconds to tolerate for clock skew when verifying exp or nbf claims.
 che.keycloak.allowed_clock_skew_sec=3
+
+#     Use the OIDC optional `nonce` feature to increase security.
+che.keycloak.use_nonce=true
+
+#     URL to the Keycloak Javascript adapter we want to use.
+#     if set to NULL, then the default used value is  
+#     `${che.keycloak.auth_server_url}/js/keycloak.js`,
+#     or `<che-server>/wsmaster/keycloak/OIDCKeycloak.js`
+#     if an alternate `oidc_provider` is used 
+che.keycloak.js_adapter_url=NULL
```

Comment:
should we add these new props to the `che.env` ?

Reply:
not sure, because its main use, for now, will be to be used on the `rh-che` side (OSIO Che) to use the `fabric8_auth` OIDC provider instead of raw Keycloak.

Reply:
I think that's fine for che 5 but you probably should add the new properties in `che.env` too for che 6.